### ISSUE TITLE
[Feature Request] Nuked-SC55

### ISSUE THREAD

**Issue by crashGG** [joncampbell123/dosbox-x#4995]
### What you want

Integrate Nuked-SC55 code as a mididevice option
https://github.com/nukeykt/Nuked-SC55

### Additional information

After many years of improvements and code iterations, Nuked-SC55 has become relatively complete and highly simulated. The effect sounds almost no different from real hardware. If dosbox-x could Integrat Nuked-SC55, a large number of games developed using the sc55 device standard will benefit from it.

### Have you checked that no similar feature request(s) exist?

- [X] I have searched and didn't find any similar feature request.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by maron2000** [comment]
Priority shouldn't be so high, since the official binary already works quite fine.

![sc55](https://github.com/joncampbell123/dosbox-x/assets/68574602/fc584aaf-70e8-4b48-ba2b-ce8f55c1671f)

**Comment by crashGG** [comment]
Yes,the official binary already works quite fine.But it requires a lot of tedious operations, especially different games need to switch to different midi devices and sound libraries.

**Comment by nickkeane** [comment]
fyi, Dosbox Staging has recently merged an audio plugin system and support for Nuked SC-55. It is currently being tested prior to release.

https://github.com/dosbox-staging/dosbox-staging/pull/4090

**Comment by rluetzner** [comment]
Hi! Sorry for barging in, but it seems like this has been forgotten. It's been almost two years now and the SC-55 CLAP seems pretty stable and works great. 

Any chance that we could get this merged?
